addon/helpers/one-way-select/contains.js

```diff
@@ -1,13 +1,19 @@
 import Ember from 'ember';
 
-const { isArray, isPresent, get } = Ember;
+const {
+  A,
```

Comment:
The issue with `A` is that JSHint complains about invoking a function starting with a capital letter without `new` keyword.